basics/the-blockchain/actors.md

```diff
@@ -1,133 +1,141 @@
-5. Using the state information, the consensus mechanism randomly awards blocks to the storage providers with the most power, and the [`RewardActor`](actors.md#rewardactor) sends FIL to storage providers.
+A basic example of how actors are used in Filecoin is the process by which storage providers prove storage and are rewarded:
+
+1. The [`StorageMinerActor`](actors.md#storagemineractor) processes proof of storage submitted by a storage provider.
+2. If valid, the storage provider is awarded storage power.
+3. The [`StoragePowerActor`](actors.md#storagepoweractor) records this power in the system.
+4. During block validation, the state recorded by the `StoragePowerActor` is checked.
+5. Using this state, the consensus mechanism rewards storage providers with the most power by awarding blocks, and the [`RewardActor`](actors.md#rewardactor) sends FIL as a reward.
 
 ### Blocks
 
-Each block in the Filecoin chain contains the following:
+Each block in the Filecoin chain contains:
 
-* Inline data such as current block height.
-* A pointer to the current state tree.
-* A pointer to the set of messages that, when applied to the network, generated the current state tree.
+- Inline data such as the current block height.
+- A pointer to the current state tree.
+- A pointer to the set of messages that, when applied to the network, generated the current state tree.
 
 ### State tree
 
-A [Merkle Directed Acyclic Graph (Merkle DAG)](../../reference/general/glossary.md#merkle-directed-acyclic-graph) is used to map the state tree and the set of messages. Nodes in the state tree contain information on:
+The _state tree_ is a [Merkle Directed Acyclic Graph (Merkle DAG)](../../reference/general/glossary.md#merkle-directed-acyclic-graph). It maps the relationships between actors and their states. Nodes in the state tree store:
+
+- Information about actors, including FIL balances and nonces.
+- Links (CIDs) pointing to actor state data.
 
-* Actors, like FIL balance, nonce, and a pointer (CID) to actor state data.
-* Messages in the current block
+The state tree is updated by messages, which trigger state transitions and ensure it reflects the latest state of the network.
 
 ### Messages
 
-Like the state tree, a Merkle Directed Acyclic Graph (Merkle DAG) is used to map the set of messages for a given block. Nodes in the messages may contain information on:
+Messages are the instructions actors use to interact with each other. These are also represented in a Merkle DAG. Nodes in the messages may contain:
 
-* The actor the message was sent to
-* The actor that sent the message
-* Target method to call on the actor being sent the message
-* A cryptographic signature for verification
-* The amount of FIL transferred between actors
+- The actor the message was sent to.
+- The actor sending the message.
+- The method to call on the receiving actor.
```

Comment:
```suggestion
- The method to call on the target actor.
```
Receiving suggests that there is a movement of tokens and not just that an actor may perform an action. 

Think of an actor as a serverless function, you target the function of a particular serverless function not have it receive a method call.